r4/src/main/java/org/opencds/cqf/r4/providers/PlanDefinitionApplyProvider.java

```diff
@@ -5,6 +5,8 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import javax.xml.bind.JAXBException;
```

Comment:
While you are in this file can you remove the unused import on line 28?
`import org.opencds.cqf.common.exceptions.NotImplementedException;`
Probably should have been removed in the PR that implemented the previously non-implemented thing but oh well

Reply:
Did this a some other simple clean up.